src/modules/ipops/doc/ipops_admin.xml

```diff
@@ -883,6 +883,57 @@ if(dns_query("test.com", "xyz"))
 
     </section>
 
+    <section id="ipops.f.ptr_query">
+      <title>
+        <function moreinfo="none">ptr_query(ip, pvid)</function>
+      </title>
+
+      <para>
+		  Store the hostname that correspond to ip
```

Comment:
s/ip/an IP address/

Also document if you support both IPv4 and IPv6 clearly. Thanks!

Reply:
Fixed